src/plans/create/steps/general-information/PlanProjectField.tsx

```diff
@@ -47,7 +43,6 @@ export const PlanProjectField: FC<PlanProjectFieldProps> = ({ onSelect }) => {
               selected={field.value}
               onSelect={(_, value) => {
```

Comment:
if u removed the custom onSelect func, i think u can omit is as field will do it

Reply:
I don't understand the ask here, but the onSelect will need `field.onChange(value);` regardless of the existence of the custom onSelect that used to be passed down into this function.

Reply:
TypeaheadSelect's onChange does not change the value for this TypeaheadSelect component, only onSelect, plus there are more operations happening in the onSelect than just an onChange event. Spreading field into components only works when the prop types match and are more basic (like a TextInput).